### ISSUE TITLE
Classification interface

### ISSUE THREAD

**Issue by brian-c** [zooniverse/Panoptes-Design#3]
Mostly looking for style here, if you think it can't be inferred from the style guide.

![screen shot 2015-02-03 at 3 25 42 pm](https://cloud.githubusercontent.com/assets/464389/6023026/03002bb8-abb9-11e4-83c5-94613b2ad6b8.png)

Subject on the left(/top), workflow task on the right(/bottom). Tasks are generally  will have a **question** and a series of **answers**. Single-choice tasks have radio button answers, multiple-choice tasks have checkbox answers. Drawing tasks' answers are a list of **tools** (like Penguins, LHC), each tool comes with a little icon (ellipse, rectangle, polygon, etc.)

Questions and answers will be markdown of any length, maybe with images or formatting.

Task navigation is back/next/done, depending on the task and the answer the user gave it.

**Comment by brian-c** [comment]
Sorry, I forgot there's a classification summary view when you're finished.

![screen shot 2015-02-03 at 6 15 16 pm](https://cloud.githubusercontent.com/assets/464389/6026202/b458e78a-abd0-11e4-9f38-0df6fe1977c4.png)

It shows each task and the answer you gave. You can expand it to see all the available choices. This'll be the thing that shows you an expert classification, and it'll be used to view aggregations when those are ready.

**Comment by heathv** [comment]
Here's what I came up with, hopefully I covered everything. I mocked it up on a white background which keeps inline with the style guide, but let me know if things should be made for a black/dark background (mostly dark gray text would become white)

![sloan-classification-ui-1](https://cloud.githubusercontent.com/assets/4551411/6027518/515f02b8-aba8-11e4-867e-5a649e60ac71.jpg)
![sloan-classification-ui-2](https://cloud.githubusercontent.com/assets/4551411/6027517/515eda18-aba8-11e4-9ce8-a52da9872c26.jpg)
![sloan-classification-ui-3](https://cloud.githubusercontent.com/assets/4551411/6027516/515ece56-aba8-11e4-846c-be07557868cb.jpg)
![sloan-classification-ui-4](https://cloud.githubusercontent.com/assets/4551411/6027515/515ea688-aba8-11e4-9624-557a211eca57.jpg)

**Comment by brian-c** [comment]
Cool, that's great, thanks.

Right now project pages have a custom background image with a dark overlay. I don't think a light overlay would work (hella eye strain). I can probably figure out inverting things if you've got stuff to do. Otherwise, one more.

**Comment by heathv** [comment]
Dark version below—general principle should apply to the other views

![sloan-classification-ui-dark](https://cloud.githubusercontent.com/assets/4551411/6028558/21feea8a-abb0-11e4-8796-4e187dc41ce9.jpg)

**Comment by brian-c** [comment]
Awesome, thanks. I should be able to get this implemented tomorrow.

**Comment by brian-c** [comment]
@heathv, just deployed to demo, have a look at these:
- http://demo.zooniverse.org/panoptes-front-end/#/projects/
- http://demo.zooniverse.org/panoptes-front-end/#/projects/brian-testing/Supernovae

Not quite finished, and there are still some obvious bugs, but if any of the liberties I've taken design-wise are unacceptable, let me know. Diagonal stripes indicate a missing image; don't freak out!

Thanks!

**Comment by ttfnrob** [comment]
This is looking really lovely :)

Great on my phone too!

**Comment by aliburchard** [comment]
Heya - the designs look awesome! I'm a bit late to the game on this one, and it looks like this (http://demo.zooniverse.org/panoptes-front-end/#/projects/brian-testing/Supernovae) has been overwritten? Looking forward to playing around with it once it's redeployed!

**Comment by brian-c** [comment]
Master is auto-deployed now, sorry.

I just deployed the restyle branch to http://demo.zooniverse.org/panoptes-front-end/restyle/#/. I don't think this'll be overwritten by the automatic deploys, but we'll see.

**Comment by heathv** [comment]
Here's a few initial comments, and I can add to this list or start a new issue as things progress towards launch.

**General/Misc**
- [ ] _About, Projects, Discuss_ - These can be inline with (and to the right of) the Zooniverse logo, can remain where they are now mobile layouts.
- [ ] _Project Nav_ - Aligned to the left — In the style guide I intended for this style to be for sub-nav within a secondary page, but it looks really clean as main nav. 

**Classification Interface**
- [ ] _Task buttons_ - I'd say solid/opaque for active buttons, transparent only for disabled buttons. 
- [ ] _Back/Next buttons_ - Move up from the base-alignment with the subject. In some instances (usually 16:9 displays) the bottom of the (placeholder) subject is below the bottom edge of the browser, pushing them out of view. [Example here, window maximized at 1280x800](https://www.dropbox.com/s/830t1zd3z04ospi/Screenshot%202015-02-06%2010.50.18.png?dl=0)
- [ ] _Fixed width?_ - On large screen resolutions (1080p on up) the task area can get really stretched and separated from the subject. Might be worth considering a max-width, or a fixed width on larger break-points.

**Comment by brian-c** [comment]
I'd like to leave the site-level nav is it is for now. In general, I'd like to pin down one design that works on big and small screens, and optimize later as needed.

Moved project nav left. Looks a little horsey. I think it's the avatar. Projects are special; want to design special secondary-level nav tabs for them?

Fixed task button colors.

I've actually removed the gap between the task and the Back/Next buttons. People are having a heck of a time dealing with the subject extending below the fold when the page loads. This should help that, and I'm now auto-scrolling to the top of the subject. We'll see how it plays.

The task area is now locked between 15 and 35 em, which in this font is between about 30 and 80 characters, which seems pretty comfortable. The actual size and position of the task area is dependent on the size of the subject. Still trying to get this optimal, but it's better now than it was.

Demo here: http://demo.zooniverse.org/panoptes-front-end/restyle/#/projects/brian-testing/Supernovae/classify
